packages/web3Connector/index.js

```diff
@@ -63,27 +63,25 @@ module.exports = async (embark) => {
 
   web3Location = web3Location.replace(/\\/g, '/');
 
-  embark.events.emit('runcode:register', '__Web3', require(web3Location), async () => {
-    const symlinkLocation = await generateSymlink(embark, web3Location);
+  const symlinkLocation = await generateSymlink(embark, web3Location);
 
-    let code = `\nconst Web3 = global.__Web3 || require('${symlinkLocation}');`;
-    code += `\nglobal.Web3 = Web3;`;
+  let code = `\nconst Web3 = global.Web3 || require('${symlinkLocation}');`;
```

Comment:
We can't do that sadly. Because `global.Web3` will be overwritten by Metamask and it uses 0.x, which is bad.
We need a way to maybe register `Web3` and `__Web3` because just `Web3` will make the dapp bug

Reply:
I see. Another reason we should try to re-separate the code run in the VM and the browser. Will change back.

Reply:
This might be affecting ENS and Whisper modules as well since they also rely on Web3.

Reply:
We can always register both. The only thing that NEEDS to be in the Dapp is something that is not `Web3`, because Metamask screws us

Reply:
All fixed up. Could you please re-review and remove changes needed if you're ok with this?